src/ui.form-date-picker-range/composables/attrs.composable.js

```diff
@@ -123,8 +105,147 @@ export default function useAttrs(props, { isShownMenu, isTop, isRight }) {
     }
   }
 
+  const variantKeys = [
+    "rangeInputFirst",
+    "rangeInputLast",
+    "periodButtonActive",
+    "periodDateWeekList",
+    "periodDateMonthList",
+    "periodDateQuarterList",
+    "periodDateYearList",
+    "firstPeriodGridDate",
+    "firstPeriodListDate",
+    "lastPeriodGridDate",
+    "lastPeriodListDate",
+    "periodDateActive",
+    "periodDateInRange",
+  ];
+
+  for (const key of variantKeys) {
+    if (key === "rangeInputFirst") {
+      const rangeInputAttrs = attrs.rangeInputAttrs;
+
+      attrs[`${key}Attrs`] = computed(() => ({
+        ...rangeInputAttrs.value,
+        class: cx([rangeInputAttrs.value.class, config.value.rangeInputFirst]),
+      }));
+    }
+
+    if (key === "rangeInputLast") {
+      const rangeInputAttrs = attrs.rangeInputAttrs;
+
+      attrs[`${key}Attrs`] = computed(() => ({
+        ...rangeInputAttrs.value,
+        class: cx([rangeInputAttrs.value.class, config.value.rangeInputLast]),
+      }));
+    }
+
+    if (key === "periodButtonActive") {
+      const periodButtonAttrs = attrs.periodButtonAttrs;
+
+      attrs[`${key}Attrs`] = computed(() => ({
+        ...periodButtonAttrs.value,
+        class: cx([periodButtonAttrs.value.class, config.value.periodButtonActive]),
+      }));
+    }
+
+    const periodDateListAttrs = attrs.periodDateListAttrs;
+
+    if (key === "periodDateWeekList") {
+      attrs[`${key}Attrs`] = computed(() => ({
+        ...periodDateListAttrs.value,
+        class: cx([periodDateListAttrs.value.class, config.value.periodDateWeekList]),
+      }));
+    }
+
+    if (key === "periodDateMonthList") {
+      attrs[`${key}Attrs`] = computed(() => ({
+        ...periodDateListAttrs.value,
+        class: cx([periodDateListAttrs.value.class, config.value.periodDateMonthList]),
+      }));
+    }
+
+    if (key === "periodDateQuarterList") {
+      attrs[`${key}Attrs`] = computed(() => ({
+        ...periodDateListAttrs.value,
+        class: cx([periodDateListAttrs.value.class, config.value.periodDateQuarterList]),
+      }));
+    }
+
+    if (key === "periodDateYearList") {
+      attrs[`${key}Attrs`] = computed(() => ({
+        ...periodDateListAttrs.value,
+        class: cx([periodDateListAttrs.value.class, config.value.periodDateYearList]),
+      }));
+    }
+
+    const periodDateAttrs = attrs.periodDateAttrs;
+
+    if (key === "periodDateActive") {
+      attrs[`${key}Attrs`] = computed(() => ({
+        ...periodDateAttrs.value,
+        class: cx([periodDateAttrs.value.class, config.value.periodDateActive]),
+      }));
+    }
+
+    if (key === "periodDateInRange") {
+      attrs[`${key}Attrs`] = computed(() => ({
+        ...periodDateAttrs.value,
+        class: cx([periodDateAttrs.value.class, config.value.periodDateInRange]),
+      }));
+    }
+
+    if (
+      [
+        "firstPeriodGridDate",
+        "firstPeriodListDate",
+        "lastPeriodGridDate",
+        "lastPeriodListDate",
+      ].includes(key)
```

Comment:
use exact comparison, includes may cause unexpected issue if some key will be added with the same prefix like `lastPeriodListDateWrapper`